config/i18n-tasks.yml

```diff
@@ -0,0 +1,181 @@
+# i18n-tasks finds and manages missing and unused translations: https://github.com/glebm/i18n-tasks
+
+# The "main" locale.
+base_locale: en
+## All available locales are inferred from the data by default. Alternatively, specify them explicitly:
+# locales: [es, fr]
+## Reporting locale, default: en. Available: en, ru.
+# internal_locale: en
+
+# Read and write translations.
+data:
+  ## Translations are read from the file system. Supported format: YAML, JSON.
+  ## Provide a custom adapter:
+  # adapter: I18n::Tasks::Data::FileSystem
+
+  # Locale files or `Dir.glob` patterns where translations are read from:
+  read:
+    - config/locales/%{locale}.yml
+    - config/locales/devise.%{locale}.yml
+    - app/components/**/*.%{locale}.yml
```

Comment:
**nitpick:** Is this line required still? I thought you removed those sidecar translations files.

Reply:
I needed to add `app/components/**/*.%{locale}.yml` in order to run `i18n-tasks normalize -p` so those files would be found for deletion. I suppose I could remove that line now, but thought it wouldn't hurt to leave it in. Would there be any benefit in leaving it, just in case locale files were added in the future? I can remove the line too. It makes no difference to me.

Reply:
It will delete them when you run normalize if they aren't used? If so in that case we could leave it in.

Reply:
It will only delete when passing -p parameter as it applies the write rules, https://github.com/glebm/i18n-tasks?tab=readme-ov-file#normalize-data. I'm assuming we want i18n-tasks to read all the locale files within irida-next.